static/js/taxpayer_creation_form_logic.js

```diff
@@ -82,45 +95,36 @@ $("#id_address_form-country").change(function(){
 function arg_is_ri(){
     $(".ri>input").prop("required", true);
     set_optionals();
-    console.log("ejecutando arg_is_ri");
-
 }
 function arg_is_mono(){
     $(".ri>input").prop("required", false);
     $(".mono>input").prop("required", true);
-    console.log("ejecutando arg_is_mono");
-
 }
 
 function arg_bank_change(){
     if ($("#id_taxpayer_form-taxpayer_condition").val() == "responsable_inscripto"){
 		$(".ri").show();
-        console.log("es ri");
         arg_is_ri();
-
     }
     else if ($("#id_taxpayer_form-taxpayer_condition").val() == "monotributista"){
         $(".ri").hide();
         $(".mono").show();
         arg_is_mono();
-        console.log("es mono");
     }
     else {
```

Comment:
Same as this [comment](https://github.com/evbeda/suppliers_management/pull/234/files#r420377067)